cmd/ceremony/cert.go

```diff
@@ -77,7 +77,7 @@ var AllowedSigAlgs = map[string]x509.SignatureAlgorithm{
 	"ECDSAWithSHA512": x509.ECDSAWithSHA512,
 }
 
-func (profile *certProfile) verifyProfile(root bool) error {
+func (profile *certProfile) verifyProfile(root bool, ocspSigner bool) error {
```

Comment:
How about an enum `certificateType` that consists of Root, Intermediate, and OCSPSigner? Then we could avoid having bool args to this function, which makes it more confusing to read at the call site. This could also replace the bool parameter to `makeTemplate`.